index.html

```diff
@@ -15,8 +15,72 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
-  <body>
+
+  
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
+      
+      <body>
+      <div class="header-container"> 
```

Comment:
Just be careful about the indentation here, `<body>` should be further to the left of `<div>`